scenario/context/CometContext.ts

```diff
@@ -40,6 +40,11 @@ import { Requirements } from '../constraints/Requirements';
 
 export type ActorMap = { [name: string]: CometActor };
 export type AssetMap = { [name: string]: CometAsset };
+export type MigrationData = {
+  migration: Migration<any>;
+  preMigrationBlockNumber?: number;
+  postMigrationBlockNumber?: number;
+}
```

Comment:
The current approach is to cache the old and new block numbers at which to grab the states.